php/Terminus/Site.php

```diff
@@ -229,7 +229,7 @@ public function notifications() {
   public function import($url) {
     $path = 'environments/dev/import';
     $drush_archive = 1;
-    if ("wordpress" == $this->information->framework) {
+    if (isset($this->information->framework) && ("wordpress" == $this->information->framework)) {
```

Comment:
Could we extract that out into a method like `Site#hasFramework("wordpress")` perhaps? Seems like we do this a lot.